apps/fastlane/helpers/build_helper.rb

```diff
@@ -0,0 +1,132 @@
+require 'json'
+require 'xcodeproj'
+require_relative 'github_helper.rb'
+
+platform :android do
+  lane :build do |values|
+    build_notes = get_build_notes()
+    test_groups = get_build_test_groups()
+    app_package_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name) # get package_name from Appfile
+
+    UI.important(find_firebase_app_id(app_identifier: app_package_name))
```

Comment:
```suggestion
    UI.important("Firebase app id: #{find_firebase_app_id(app_identifier: app_package_name)}"))
```

This code doesn't compile, but trying to illustrate a suggestion of labeling what the value is so it's easier to read in CI logs.

Reply:
I like the comment you made later on, `# Firebase app id is required. Get it from google-services.json file`. perhaps we can include that here, too?